demos/index.html

```diff
@@ -54,7 +55,7 @@
 
                 <div id="floec-chartAuthoring"></div>
 
-                <script type="text/javascript">
+                <script >
```

Comment:
Remove the extra space in btw ```script``` and ```>```. Please also fix the indentation at line 60.